problems/unsuccessful_products.md

```diff
@@ -0,0 +1,79 @@
+---
+slug: unsuccessful-products
+stages:
+  - development
+short_description: About 30 to 45% of new products fail to deliver any meaningful financial return. There are many reasons why this happens. 
+tags:
+  - Unsuccessful Products
+  - market
+  - timing
+  - competition
+  - support
+keywords:
+  - market
+  - timing
+  - competition
+  - support
+  - price
+  - promotion
+---
+
+# Unsuccessful Product
+
+## TL;DR
+
+About 30 to 45% of new products fail to deliver any meaningful financial return. This typically happens due to a number of reasons – poor product / market fit, high price, failure to understand customer needs, fixing a non-existing problem, etc. 
+
+## Why Do Some Products Fail? 
+
+#### Poor Product Quality
```

Comment:
Please, rewrite this part from Poor Product Quality to Failure to Understand Consumer Needs and Wants. It is too similar.

Poor product quality: Obviously, a product, which is of poor quality, cannot be sold in the market.

Higher price: Another reason for the failure of certain products is the price factor. Higher production and distribution costs may lead to a higher price. Such a product cannot be sold in a market consisting of middle and lower-income buyers.

Poor timing: It is important that a product, to be successful, is introduced in the market at the correct time. If it is introduced at an unsuitable time it may turn out to be a failure.

Inherent defect: There may be an inherent defect in the product, which may affect its market potentialities. Such a product may not be preferred by the buyers even if the defect is rectified later.

 Lack of promotional measures: Popularizing the brand, particularly, in the introduction stage of a product is essential. Such a step will ensure repeated buying and bring long-term benefits for the marketer. Failure to do so will ‘prove to be disastrous for the product.

Reply:
Wording changed.